CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 ### Fixed
 
 - [#1070](https://github.com/improbable-eng/thanos/pull/1070) Downsampling works back again. Deferred closer errors are now properly captured.
+- [#1074](https://github.com/improbable-eng/thanos/pull/1074) Fixed fetching go-bindata failed.
```

Comment:
This is really related to development strictly, so we can remove this from CHANGELOG (:

Reply:
Done